### DISCUSSION TITLE
Question about the max size of repo

### DISCUSSION THREAD

**Discussion by Kyumi77** [community/community#123337] [New to GitHub]
Hello guys, i was wondering if there is any problem while repo between 30-40 gb files ? I mean we are working on a new unreal engine project and will use github for push and pull. What is the maximum limit of repo on free account and if there is a maximum size of repo, what it can be on Github team priced account. Waiting for your messages, thanks for now.

**Comment by davevad93** [comment]
Hi @Kyumi77 ,

Here are a couple of links from GitHub docs that address this topic:

- [About large files on GitHub](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github)
- [About Git Large File Storage](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage)

**Comment by Kyumi77** [comment]
I just read it but i didnt understand that can i repo 30 gb unreal engine project or not..

**Reply by davevad93** [reply]
I suppose you could push the project to GitHub, but only incrementally since GitHub has a maximum 2 GB limit for a single push. 
The thing is I've never pushed to GitHub such a big project so I can't really say if it is really feasable or not.
